cucumber/authenticators/features/authn_oidc.feature

```diff
@@ -136,7 +136,7 @@ Feature: Users can authneticate with OIDC authenticator
     Then it is unauthorized
     And The following appears in the log after my savepoint:
     """
-    Errors::Conjur::RequiredResourceMissing
+    Errors::Authentication::Security::AccountNotDefined
```

Comment:
I think you might need to update the other `Scenerios` to accept a valid account (something like "I authenticate via OIDC with id token and account "existing"". Otherwise, they are all failing on `AccountNotDefined` when that is what we want to check in other tests

Reply:
good catch.

Reply:
but just to clarify - the file that you are referring to is an integration test. The failing tests were UTs.

Reply:
You do not need to fetch the ID Token in the beginning of the scenario

Reply:
cool, removed it from the test.